src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,72 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.menu {
```

Comment:
https://prnt.sc/L3Ekd4-d3EEY => Just add z-index and position to fix it)

https://prnt.sc/48UHyMaZ20aU => add cross icon (I can click, but I can't see an icon)